contrib/babelfishpg_tsql/sql/babelfishpg_tsql.sql

```diff
@@ -983,10 +983,10 @@ AND has_table_privilege(t1.oid, 'SELECT,INSERT,UPDATE,DELETE,TRUNCATE,TRIGGER');
 GRANT SELECT ON sys.sp_tables_view TO PUBLIC;
 
 CREATE OR REPLACE FUNCTION sys.sp_tables_internal(
-	in_table_name sys.nvarchar(384) = '',
```

Comment:
I would assume this issue also extends to other sp_tables params. Did we see if the same handling of '%' for table_qualifier would also works for others?

Reply:
Actually this `'%'` case is already handled for other params and test cases for the same are already there in other test file.